README.md

```diff
@@ -11,7 +11,9 @@ In `mix.exs`, add the ExMachina dependency:
 
 ```elixir
 def deps do
-  [{:ex_machina, "~> 0.4"}]
+  [
+    {:ex_machina, "~> 0.4", only: "test"}
+  ]
```

Comment:
Yeah I think condensed is preferred for now, but we might change it in the future. I'll have to think about it some more.